### ISSUE TITLE
[alpha 6] navbar display small when collapse

### ISSUE THREAD

**Issue by vutaviva** [twbs/bootstrap#21723]
I try navbar in version alpha 6. When I change windows size to small, collapse button appear. However, row height is changed to small. It does not cover collapse button (when there is no navbar-brand element).

```html
<nav class="navbar navbar-toggleable-md navbar-light bg-faded">
  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
    <span class="navbar-toggler-icon"></span>
  </button>

  <div class="collapse navbar-collapse" id="navbarNavDropdown">
    <ul class="navbar-nav">
      <li class="nav-item active">
        <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
      </li>
      <li class="nav-item">
        <a class="nav-link" href="#">Features</a>
      </li>
    </ul>
  </div>
</nav>
```

**Comment by mdo** [comment]
You need something to set the `display` to `flex` here. Navbars require a `.container` to do that, so if you opt to remove the container, you'll need to manually address that.

**Comment by iatek** [comment]
I think this will be fixed with #22230
